homework-submission/React_HW1/index.html

```diff
@@ -0,0 +1,155 @@
+        },
+        {
+          description: '* Eat breakfast',          
+          deadline: 'Fri Sep 15 2017',
+          done: ''
+        }
+      ];
+      
+         const todoListDynamic = [
+        {
+          id: 1,
+          description: 'Get out of bed',
+          deadline: '2017-09-11',
+          done: true,
+        },
+        {
+          id: 2,
+          description: 'Brush teeth',
+          deadline: '2017-09-10',
+          done: false,
+        },
+        {
+          id: 3,
+          description: 'Eat breakfast',
+          deadline: '2017-09-09',
+          done: false,
+        },
+      ];
+
+// TableTitle Component used by both static and dynamic todo list
+      const TableTitle = () => {
+        return (
+          <thead>
+            <tr>
+              <th>Description</th>
+              <th>Deadline</th>
+              <th>Done</th>
+            </tr>
+          </thead>
+        );
+      }
+
+      class StaticTodoContent extends React.Component {
+        render () {
+          return (
+            <tbody>
+              <tr>
+                <td>{todoListStatic[0].description}</td>
+                <td>{todoListStatic[0].deadline}</td>
+                <td>{todoListStatic[0].done}</td>
+              </tr>
+              <tr>
+                <td>{todoListStatic[1].description}</td>
+                <td>{todoListStatic[1].deadline}</td>
+                <td>{todoListStatic[1].done}</td>
+              </tr>
+              <tr>
+                <td>{todoListStatic[2].description}</td>
+                <td>{todoListStatic[2].deadline}</td>
+                <td>{todoListStatic[2].done}</td>
+              </tr>
+            </tbody>
+          );
+        }
+      }
+
+
+// Static todo list used TableTitle component
+      class StaticTodo extends React.Component {
+        render () {
+          return (
+            <div>
+              <h2> Todo list as static</h2>
+              <table>
+                <TableTitle />
+                <StaticTodoContent />
+              </table>
+            </div>
+          );
+        }
+      }
+
+// ContentDynamicTodo component is for map function and used props
+      const ContentDynamicTodo= ({ description, deadline, className, done } )=> {
+        return (
+          <tbody>
+            <tr className={className}>
+              <td>{description}</td>
+              <td>{deadline}</td>
+              <td>{className}</td>
+            </tr>
+          </tbody>
+        );
+      };
+      
+// Dynamic todo list used map function component (ContentDynamicTodo) and TableTitle component 
+      const DynamicTodo = () => {
+        return (
+          <div>
+            <h2>Todo list as dynamic</h2>
+           <table>
+              <TableTitle />
+              {todoListDynamic.map(({ id, description, deadline, done }) => ( 
+                <ContentDynamicTodo key={id} description={description} deadline={deadline} className={done ? '√' : 'x'}/>))
```

Comment:
In line 114 you are using className as `'√'` Or `'x'` which make it a little tricky to catch and follow, I would prefer using English clear words instead, and for line 117 it's looks with styling Great 💯

Reply:
I did not understand what you mean hear?